wasm-app/src/components/atoms/ModalConfirm.tsx

```diff
@@ -0,0 +1,45 @@
+import { List, ListItem } from "@mui/material";
+import { Modal, ModalBody, ModalHeader, Typography } from "@skuber/components";
+import { defaultTheme } from "@skuber/theme";
+import { ModalFooter } from "./ModalFooter";
+
+type ModalConfirmProps = {
+  open: boolean;
+  onClose: () => void;
+  onConfirm: () => void;
+  title: string;
+  description: string;
+  descriptionDetails: string[];
+};
+
+export const ModalConfirm = ({ open, onClose, onConfirm, title, description, descriptionDetails }: ModalConfirmProps) => {
+  return (
+    <Modal width={434} open={open} onClose={onClose}>
+      <ModalHeader title={title} onClose={onClose} />
+      <ModalBody>
+        <Typography>{description}</Typography>
+        {descriptionDetails && descriptionDetails.length > 0 && (
+          <List
+            sx={{
+              paddingLeft: "24px",
+              listStyleType: "disc",
+              ...defaultTheme.typography.body2,
```

Comment:
```suggestion
              typography: "body2",
```
chị comment mấy lần rồi

Reply:
Chỗ này của List đâu dùng được chị nhỉ